### PR TITLE
Image encoding

### PR DESCRIPTION
**Pull request by browna38**
During testing on the master branch we found issues with the video capturing and saving. We also found that an improvement could be made to the csv file format and frequency of sample points. Updates and upgrades have been made and are ready to be tested on hardware.

### CODE REVIEW

File: VideoCapture.py [hunk 1]
```diff
@@ -55,15 +58,16 @@
...
         text = text + center
         cv2.putText(frame, text, (x, y - 10),
             cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 255), 2)
             # if the barcode text is currently not in our CSV file, write
             # the timestamp + barcode to disk and update the set
-        if barcodeData not in found:
-            csv.write("{},{}\n".format(datetime.datetime.now(),
-                barcodeData,x,w,y,h))
+        timenow = datetime.datetime.now()
+        if barcodeData == 'Aurora Aerial':
```
Comment (StephenVDK) [inline VideoCapture.py]: Can we put this in a global variable so it's easier to access?

Comment (StephenVDK) [reply VideoCapture.py]: Like expectedData = 'Aurora Aerial' in a header or at the top?

Comment (browna38) [reply VideoCapture.py]: Yes

File: VideoCapture.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
 	help="path to output CSV file containing barcodes")
 args = vars(ap.parse_args())
 
+exceptedDate = 'Aurora Aerial'
```
Comment (StephenVDK) [inline VideoCapture.py]: expected date?

Comment (browna38) [reply VideoCapture.py]: oh dear, good catch

**Pull request merged**